package.json

```diff
@@ -145,6 +145,8 @@
     "piwik-react-router": "0.8.2",
     "popper.js": "1.16.1",
     "pouchdb-adapter-cordova-sqlite": "^2.0.7",
+    "pouchdb-adapter-idb": "7.2.2",
+    "pouchdb-adapter-indexeddb": "7.2.2",
```

Comment:
Do we really need this package? Since https://github.com/cozy/cozy-client/commit/94ce6df5797ec09a436b50723aef11bdd1e9dce6#diff-a8d46e44d0bdbf56926cd82f0cd633678572de99cc8e8eeb1d56c205b3dd031eR17-R18 we updated Pouch. This adapter doesn't come by default?

Reply:
I tried to build with this version of cozy-client but I get `Error: Can't resolve 'pouchdb-adapter-indexeddb'`, so it seems necessary